src/Pages/LogIn/components/SignIn.js

```diff
@@ -0,0 +1,273 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const SignIn = props => {
```

Comment:
```suggestion
const SignIn = ({ isModal }) => {
```
이런식으로 비구조화할당 해주면 따로 변수선언을 해주지 않아도 됩니다 :)

Reply:
props로 전달받는 인자가 여러개라고하면... ({isModal, handleModalWindow}) 이런식으로 구조분해 할당할 수 있을까요?